src/alfasim_sdk/alfasim_sdk_api/api.h

```diff
@@ -812,6 +812,51 @@ DLL_EXPORT int get_liq_liq_flow_pattern_input_variable(
     void* ctx, double* out, const char* var_name, int phase_id
 );
 
+/*!
+    Gets the wall names and wall properties for a given control volume. The wall names will be given
```

Comment:
Did you check if this documentation is already being shown in ALFAsim-SDK's docs?
it seems not: https://alfasim-sdk--398.org.readthedocs.build/en/398/plugins/99_1_solver_api_reference.html#alfasim-s-solver-data

You have to go to the docs files and add a line to point to this function. In this way, you will see appearing the documentation.

Reply:
done :D